docs/azure_jumpstart_ag/contoso_supermarket/freezer_monitor/_index.md

```diff
@@ -4,4 +4,261 @@ weight: 100
+  - Section 3
+    - Under __Folder__ type _Alerts_ and press __Enter__
+    - Under __Evaluation group__ type _Alert Group_ and press __Enter__
+
+      ![Grafana showing the Folder selection](./img/grafana_add_alert_rule_folder.png)
+  - Scroll to the top of the page and click __Save and exit__
+
+    ![Grafana showing the Save and exit button](./img/grafana_add_alert_rule_save.png)
+- View your alert
+  - Back on the __Alert rules__ page, type _freezer_ in the __Search__ box and press __Enter__
+  - Expand the __Alerts > Alert Group__ folder to see your __Freezer too warm - food at risk__ alert
+  - Expand the alert to view to __Silence__ it, __Show state history__, or quickly __edit__ or __delete__ the alert
+
+    ![Grafana showing the new alert rule](./img/grafana_new_alert_rule.png)
+
+### Scenario 3: Follow the data from the freezer to the dashboards
+
+In order to troubleshoot problems with the IoT data flow, it's important to understand the components involved. Take a look at how the data flows from a simulated freezer to the dashboards in ADX and Grafana.
+
+#### MQTT Simulator
+
+The first component, which generates the data for both dashboards is the MQTT Simulator. The simulator is a Python script that runs in each AKS Edge Essentials cluster. It generates simulated temperature and humidity data for two freezers in each environment and sends the data via the MQTT protocol to the MQTT Broker.
+
+To see data being produced by the MQTT Simulator
+
+From the Client VM:
+
+- Open __Visual Studio Code__
+- Click on the __Kubernetes__ icon in the Activity Bar on the left
+
+  ![Visual Studio Code showing the Kubernetes icon](./img/vscode_kubernetes_icon.png)
+- Right-click on the __Chicago__ cluster and select __Set as Current Cluster__
+
+  ![Visual Studio Code showing the Set as Current Cluster menu](./img/vscode_set_as_current_cluster.png)
+- Expand __Chicago__ > __Namespaces__, right-click on __sensor-monitor__ and select __Use Namespace__
+
+  ![Visual Studio Code showing the Use Namespace menu](./img/vscode_use_namespace.png)
+- Expand __Chicago__ > __Workloads__ > __Pods__ then right-click on the 'sensor-monitor-simulator-xxx' pod and select 'Logs'
+
+  ![Visual Studio Code showing the Logs menu](./img/vscode_simulator_logs_menu.png)
+- In the Logs view click __Run__ to see the logs
+
+  ![Visual Studio Code showing the Logs window](./img/vscode_simulator_logs_run.png)
+- This won't show you the values being produced, but it will show you that data is being published to the MQTT Broker
+
+  ![Visual Studio Code showing the Simulator logs](./img/vscode_simulator_logs.png)
+
+#### MQTT Broker
+
+The MQTT Broker is a container running Mosquitto in each AKS Edge Essentials cluster like the simulator. It receives the data from the simulator and sends it to the Azure IoT Hub. It also makes the data available for a third service, MQTT2Prometheus, and is explained below.
```

Comment:
Not just on each AKS Edge Essentials cluster, also on AKS

Reply:
fixed